openwillis/measures/video/preprocess_video.py

```diff
@@ -765,6 +758,7 @@ def preprocess_face_video(
 
     bb_dict = {}
     facedata_df = pd.DataFrame()
+    bbox_cols=config["bbox_dict_cols"]
 
     try:
         face_data_across_frames, fps, num_frames_vid = load_facedata_from_video(
```

Comment:
`num_frames_vid` is now deprecated so I guess we should remove it as an output from `load_facedata_from_video`

Reply:
removed in latest commit